### DISCUSSION TITLE
New-WinGetSource Help

### DISCUSSION THREAD

**Discussion by myokeeh** [microsoft/winget-cli-restsource#197] [Q&A]
I'm following the steps here: https://github.com/microsoft/winget-cli-restsource/blob/main/Tools/PowershellModule/doc/new-winget-rest-source-azure.md

I'm getting stuck here. How do I define or obtain what these values are supposed to be?

```
cmdlet Test-AzResourceGroupDeployment at command pipeline position 1
Supply values for the following parameters:
(Type !? for Help.)
manifestCacheEndpoint: !?
manifestCacheEndpoint
manifestCacheEndpoint: test1234
monitoringTenant: test1234
monitoringRole: test1234
monitoringMetricsAccount: test1234
```

**Comment by myokeeh** [comment]
@denelon, can you help?

**Reply by denelon** [reply]
@RDMacLachlan is this a known problem?

**Reply by myokeeh** [reply]
I'm working against the clock because of the imminent Microsoft Store for Business shutdown. Can you help?

**Reply by RDMacLachlan** [reply]
> I'm working against the clock because of the imminent Microsoft Store for Business shutdown. Can you help?

Hi @myokeeh,
Can you help me to understand at what step you are encountering the above message?

**Reply by myokeeh** [reply]
i get the message on my fist post after a few minutes after running the following with my own parameters. I see some of the services were created in Azure, but unsure where its getting hung up exactly.

PS C:\> New-WinGetSource -Name "contosorestsource" -ResourceGroup "WinGet" -Region "westus" -ImplementationPerformance "Demo" -ShowConnectionInstructions

**Reply by RDMacLachlan** [reply]
Hi @myokeeh,
I believe I understand what your challenge is. I'm putting together an updated release item that can be used instead of the currently available release artifact.

**Reply by myokeeh** [reply]
@RDMacLachlan any updates about this? I would like something to test before time runs out.

**Reply by RDMacLachlan** [reply]
Hi Myokeeh,
There are still some pending changes that need to be completed. I will do my best to get a new artifact (PowerShell) released this week.

**Reply by myokeeh** [reply]
I'm glad the [retirement of Store for business has been postponed](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/update-to-endpoint-manager-integration-with-the-microsoft-store/ba-p/3585077), but I want to keep evaluating options.

Also, is there a way for me to actually specify the names of the resources being spun up?

**Reply by myokeeh** [reply]
@RDMacLachlan, @denelon do you have an update?

**Reply by RDMacLachlan** [reply]
We are still working on getting this included into the released artifacts.

**Reply by myokeeh** [reply]
The Store app has been updated recently to gut our ability get LOB apps installed. We officially don't have a way to distribute our LOB apps anymore. Any updates on this?

**Reply by myokeeh** [reply]
Has this moved at all?

**Reply by RDMacLachlan** [reply]
The problem with the build pipeline has been fixed, and we are now working on having an update reviewed and then published.

**Reply by myokeeh** [reply]
@RDMacLachlan what's the status?

**Reply by RDMacLachlan** [reply]
Hi @Myokeeh, the PowerShell has been updated, and aligns with the instructions on how to setup a REST with the PowerShell module.
